composer.json

```
@@ -5,7 +5,8 @@
 	"scripts": {
 		"cs:check": "php-cs-fixer fix --dry-run --diff",
 		"cs:fix": "php-cs-fixer fix",
-		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l"
+		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l",
+		"test:unit": "phpunit -c phpunit.xml --fail-on-warning"
```

My personal favorite:
```suggestion
		"test:unit": "phpunit -c phpunit.xml --fail-on-warning --fail-on-risky --color"
```